tests/containers/docker_compose.pm

```diff
@@ -42,8 +42,14 @@ sub run {
     add_suseconnect_product(get_addon_fullname('python2')) if is_sle('=15-sp1');
 
     record_info 'Test #1', 'Test: Installation';
-    zypper_call("in docker-compose");
-    assert_script_run 'docker-compose --version';
+    if (script_output('zypper -n in docker-compose', proceed_on_failure => 1) =~ /nothing provides python3-docker/) {
```

Comment:
```suggestion
    if (script_output('zypper -n se python3-docker', proceed_on_failure => 1) != 0) {
```

Reply:
Maybe we should still consider using `zypper_call`.

Reply:
> Maybe we should still consider using `zypper_call`.

done